compiler/src/dmd/semantic3.d

```diff
@@ -314,7 +314,7 @@ private extern(C++) final class Semantic3Visitor : Visitor
         fds.checkInContractOverrides();
 
         // Remember whether we need to generate an 'out' contract.
-        immutable bool needEnsure = FuncDeclaration.needsFensure(funcdecl);
+        immutable bool needEnsure = needsFensure(funcdecl);
```

Comment:
Just a small note that D does have UFCS, and it is not wrong to take advantage of it.

Reply:
Good catch. Cheers.